src/components/ContextualHost/ContextualHost.ts

```diff
@@ -34,6 +34,8 @@ namespace fabric {
 
   export class ContextualHost {
 
+    public static hosts: Array<ContextualHost>;
```

Comment:
Just making a note that in the future, it'd be good for us to have JSDoc comments explaining these properties.